lib/model/autocomplete.dart

```diff
@@ -289,6 +289,12 @@ class MentionAutocompleteQuery {
     // TODO(#236) test email too, not just name
     // TODO(#237) test with diacritics stripped, where appropriate
```

Comment:
This TODO about diacritics belongs inside `_testName`, since it's about a change we would make within that function.